domain/src/main/java/me/androidbox/domain/alarm_manager/AlarmItem.kt

```diff
@@ -0,0 +1,8 @@
+package me.androidbox.domain.alarm_manager
+
+import org.threeten.bp.ZonedDateTime
+
+data class AlarmItem(
+    val dateTime: ZonedDateTime,
+    val message: String?
```

Comment:
Is it ok to have a null message here? I am thinking there should always be a message set. So maybe this might not be necessary

Reply:
Does every notification have a message? Depends if having a description for an agenda item is optional or not in your app

Reply:
@philipplackner I guess there will always be a message related to the notification.